.travis.yml

```diff
@@ -4,9 +4,18 @@ rust:
   - stable
   - beta
   - nightly
-#matrix:
-#  allow_failures:
-#    - rust: nightly
 script:
   - cargo build --verbose --all
   - cargo test --verbose --all
+matrix:
+#  allow_failures:
+#    - rust: nightly
```

Comment:
can you remove these comments as well? They are no longer needed anyway. We need to build on nightly.

Reply:
Yeah sure 😉 should i remove the other rust build targets (stable, beta)?